templates/graphDebug.html

```diff
@@ -4,14 +4,26 @@
     <meta charset="utf-8">
     <title><%= name %></title><% _.forEach(scripts, function (path) { %>
     <script src="<%= path %>"></script><% }); %>
-    <script src="https://cdn.rawgit.com/rtc-io/rtc/v3.1.1/dist/rtc.js"></script>
+    <script src="https://app.flowhub.io/node_modules/rtc/dist/rtc.js"></script>
     <script src="<%= noflo %>"></script>
     <style>
     #flowhub_debug_url.nodebug {
       display: none;
     }
     #flowhub_debug_url.debug {
-      display: inherit;
```

Comment:
![screenshot 2017-01-06 at 16 20 28](https://cloud.githubusercontent.com/assets/3346/21722257/17ffbfae-d42c-11e6-82e7-3d4e369879d4.png)

I had to modify it as previously it was easily rendered _under_ the other HTML. But could slightly improve how it looks while at it...